exporter/collector/integrationtest/testdata/fixtures/metrics/untyped_gauge_expect.json

```diff
@@ -691,4 +691,4 @@
       }
     ]
   }
-}
+}
```

Comment:
did you run `make fixtures` to update this file? I got the newline added back and I think that will fix the failing test

Reply:
My editor does this a lot when I open the file again 🤦 . Ran `make fixtures` again and that seems to have fixed this